### ISSUE TITLE
Mangadex is not working (Http error 503)

### ISSUE THREAD

**Issue by heartlog** [tachiyomiorg/tachiyomi-extensions#13442]
### Source information and language

Mangadex 1.3.167

Website status is showing down on tachiyomi (in webview also), but it's working fine on the other browser(ex.Chrome,Firefox).

Status is showing
**Temporary maintenance**

Even after cleaning cookies, it don't change at all.
Please, do something about it...

Thanks,

# Official reference 
Last maintenance/service updates:** 2 days ago**
[Official announcement 2 days ago](https://twitter.com/MangaDexRE?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor)
*I think they included some anti tachiyomi update during this recent maintanence.

### Expected behavior

The extension should show results ?

### Actual behavior

It should show results, not the error.

### Tachiyomi version

0.13.6

### Android version

Android 12

### Other details

# Extension search interface

![Screenshot_20220911-223340_Tachiyomi](https://user-images.githubusercontent.com/76758776/189540228-0e58ed90-6ff4-4208-80ee-f6dc5ae87875.jpg)

# webview

![Screenshot_20220911-223349_Tachiyomi](https://user-images.githubusercontent.com/76758776/189540276-918d1540-619e-4713-9383-beda7e8d65ef.jpg)

# On browser 
**Search mangadex.org**

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by CrepeTF** [comment]
Tachiyomi has been temporarily been blocked by MangaDex due to ongoing server bugs. No ETA can be estimated at this time due to the cause of the bug being unknown.

**Comment by vaishnavm217** [comment]
Weird thing is "open webview" doesn't load the website and gives api error, but opening the same link in browser works

**Comment by kevin01523** [comment]
@heartlog just run the back  from settings and import it to the mangadex tachi fork since one that works basically migrate mangadex one on that app it only imports the mangadex enteies so its a smooth migration
